airbyte-webapp/src/components/connectorBuilder/Builder/UiYamlToggleButton.module.scss

```diff
@@ -0,0 +1,36 @@
+@use "scss/variables";
+@use "scss/colors";
+
+.button {
+  cursor: pointer;
+  border: variables.$border-thin solid colors.$dark-blue;
+  background-color: colors.$dark-blue;
+  display: flex;
+  border-radius: 6px;
+  padding: 0;
+  overflow: hidden;
+  height: 21px;
+  width: 96px;
+  position: absolute;
+  top: 15px;
+  left: 52px;
+}
+
+.text {
+  height: 100%;
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-weight: 700;
+}
+
+.selected {
+  background-color: colors.$white;
```

Comment:
Maybe it's just me, but it feels like these are flipped. I perceive the darker one as the "selected" one

Reply:
I actually felt that way too when I first added this, and convinced myself that the designer knew better. But since you have also voiced this, I think I'll swap them